modules/the-open-data-hub-user-interface.adoc

```diff
@@ -62,7 +62,7 @@ Click the *Launch application* button on an application card to open the applica
 
 Applications -> Explore:: The *Explore* page displays applications that are available for use with {productname-short}.
 Click on a card for more information about the application or to access the *Enable* button.
-The *Enable* button is only visible if your administrator and enabled an application at the {productname-short} level.
+The *Enable* button is only visible if an application does not require an OpenShift operator installation. 
```

Comment:
A couple of suggestions:

- "The *Enable* button is visible only..."
- When we develop some style standards, I'm sure we'll end up agreeing on 'Operator' (capitalized). I've already applied this in new documentation. It might make sense to do the same here.

Reply:
thanks!